docs/integrations.md

```diff
@@ -11,6 +11,20 @@ The following instructions use [Semgrep CI](https://github.com/returntocorp/semg
 !!! danger
     `SEMGREP_APP_TOKEN` is a secret value: DO NOT HARDCODE IT AND LEAK CREDENTIALS. Use your CI provider's secret or environment variable management feature to store it. 
 
+<details><summary>Generic CI Provider</summary>
```

Comment:
From a search perspective, do you think people will discover this section when they're looking for instructions for "non-supported" CI providers?

Reply:
@dlukeomalley How about adding something like, “For Buildkite, CircleCI, [ insert other popular CI provider names], add the following to your workflow:” ?

That way when people search for their particular CI provider by name, this will appear in the results.